modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -128,7 +129,7 @@ protected function getColsFromFile(string $file_path, string $delimiter): array
     $f = fopen($file_path, 'r');
 
     // Ensure the file could be successfully opened.
-    if ($f === FALSE) {
+    if (!isset($f) || $f === FALSE) {
```

Comment:
I think this is 2.18 cruft that snuck back in due to me working in 2.18.   I will fix this and the others of similar sneakiness.